LSP.sublime-settings

```diff
@@ -365,16 +365,8 @@
         "syntaxes": ["Packages/PHP/PHP.sublime-syntax"],
         "languageId": "php"
     },
-    "eslint":
-    {
-      "command": ["node", "/Users/tomv/Projects/tomv564/vscode-eslint/eslint-server/src/server"],
-      "scopes": ["source.js"],
-      "syntaxes": ["Packages/Babel/JavaScript (Babel).sublime-syntax", "Packages/JavaScript/JavaScript.sublime-syntax"],
-      "languageId": "javascript",
-      "initializationOptions": {
-        "nodePath": "/usr/local/bin/node"
-      }
-    },
+    //"eslint": { },
```

Comment:
This is potentially a breaking change if someone was using existing config as a base and just changed command & nodePath but oh well, I think it's fine.